data/jobs/2022-05-15-medchemica-med-chemist.yaml

```diff
@@ -0,0 +1,27 @@
+role: Researcher in design of compounds and coordination of design-make-test-analyse cycle to drive antiviral discovery programs
+title: Medicinal Chemist
+institution: MedChemica Consultancy Ltd
+location: United Kingdom
+investigator: Dr. Ed Griffen
+investigator_url: www.medchemica.com
+contact_email: contact@medchemica.com
+supported_projects_and_cores:
+  - Project 5: Lead Optimization
+duration: 1 year initial contract with opportunity for further permanent position
+availability: September  - December 2022
+status: open
+summary: |
+  [MedChemica](http://www.medchemica.com) is leading the optimisation of new antiviral agents within the ASAP antiviral drug discovery (AViDD) Center. Within our team, we analyse biological and ADMET testing data, and combine that with protein-ligand structural data, FEP calculations and machine learning methods to both design and prioritise compounds for synthesis.  As a member of the lead optimisation team, you will work closely with teams of synthetic chemists, other medicinal chemists, computational chemists, virologists, pharmacokineticists and physical chemists to make sure the best compounds get made and tested.  The lead optimisation team is central in delivering the overall Center goals of generating new anti-virals for human trials ready to treat future pandemics.  You will be working within MedChemica Consultancy Limited where you will be a supportive and supported team member, and where you will also have opportunities to contribute to the development of new AI tools for medicinal chemistry.
```

Comment:
Can we set a consortium wide standard of American spelling :)

Reply:
Good idea!